### PR TITLE
Add AUR badge

### PR DESCRIPTION
**Pull request by eqrx**
**What this PR does / why we need it**:
Add a badge to README.md displaying the current AUR version. This makes it easier for maintainers to verify that the AUR package installs the correct KubeOne version and users can access the AUR page directly by clicking on it.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
None

**Special notes for your reviewer**:
Nope

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,11 @@
   </a>
   <a href="https://bestpractices.coreinfrastructure.org/projects/2934"><img src="https://bestpractices.coreinfrastructure.org/projects/2934/badge">
   </a>
+  <a href="https://aur.archlinux.org/packages/kubeone">
+    <img src="https://img.shields.io/aur/version/kubeone.svg" alt="AUR version" />
+  </a>
+
+https://img.shields.io/aur/version/kubeone.svg
```
Comment (kron4eg) [inline README.md]: is this an accident?

Comment (eqrx) [reply README.md]: jup, sorry

**Comment by kron4eg** [comment]
/lgtm
/approve
/hold

hold before gomod PR is in

**Comment by kron4eg** [comment]
/hold cancel

oh well, let it be. seems gomods will be a bit late.

**Pull request merged**